Description: Use private dir for libntpc
 libntpc.so is only used by the Python library.  In NTPsec 1.2.0 (when it
 first appeared), upstream is installing to LIBDIR/ntp.  However, the code
 looks for it in PYTHONDIR and the system library search paths (e.g.
 /usr/lib and /usr/lib/<triplet>).  The proposed (but not yet committed) fix
 for this is to move it LIBDIR.  I have chosen to bring in that patch (since
 it looks like that will be accepted) so the library is installed to LIBDIR
 (which I am setting to /usr/lib/<triplet> in debian/rules) and then use the
 python3-ntp.install file to move it to LIBDIR/ntp.  This patch is then
 necessary to search that location.  Keeping it a private library in
 LIBDIR/ntp avoids the need for full Debian library packaging.  This can be
 revisited once upstream clarifies whether this is intended to be a public
 or private shared library.
Forwarded: https://gitlab.com/NTPsec/ntpsec/-/merge_requests/1185#note_448678013
Origin: vendor
Author: Richard Laager <rlaager@debian.org>
Last-Update: 2020-11-16

--- a/pylib/ntpc.py
+++ b/pylib/ntpc.py
@@ -35,6 +35,8 @@
     if ntpc_path:
         ntpc_paths.append(ntpc_path)
 
+    ntpc_paths.append("@LIBDIR@/ntp/" + (_fmt() % LIB))
+
     return _dlo(ntpc_paths)
 
 
